drivers/usb_c/tcpc/ucpd_stm32.c

```diff
@@ -1027,10 +1027,9 @@ static int ucpd_transmit_data(const struct device *dev,
 /**
  * @brief Tests if a received Power Delivery message is pending
  *
- * @retval true if message is pending, else false
+ * @retval 1 if message is pending, else 0
```

Comment:
I think this can stay true and false

Reply:
Sorry, I see why you changed this to 1 and 0. I think there should be a better way to handle these return values and error codes. Because an error code can result, maybe we just stick with convention and return 0 on success and a negative error code on failure for the functions you've modified in this CL. This would mean all the places these functions are used; those return value checks would need updating.

Reply:
Hm, I think about adding the additional parameter which will be a pointer to bool that could contain the status of pending message. I don't think that using the "there's no pending message" as error code is valid, since it's not an error. However, I see that the _tcpc_receive_data_ is using the concept of returning negative values as errors and in case of success, the zero of other positive number. So it would be according to this concept of returning either negative values, or 0/1 for value got.